### PR TITLE
sbt 1.2.1

### PR DESCRIPTION
**Pull request by sullis**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -53,7 +53,7 @@ libraryDependencies ++= {
 }
 
 // configure github page
-enablePlugins(SphinxPlugin, SiteScaladocPlugin, GhpagesPlugin)
+enablePlugins(SphinxPlugin, SiteScaladocPlugin, GhpagesPlugin, SbtPlugin)
```
Comment (muuki88) [inline build.sbt]: Does this make the `sbtPlugin := true` obsolete?

**Comment by muuki88** [comment]
Can you run `sbt:scalafmt` and recommit?

```
com.lucidchart.sbt.scalafmt.ScalafmtCheckFailure: 1 Scala source not formatted in sbt-native-packager:sbt
```

**Comment by muuki88** [comment]
Thanks for pointing that out 🤗 we'll wait until a fix has been released

**Comment by muuki88** [comment]
@sullis can you upgrade to `1.2.1` ?

**Comment by sullis** [comment]
ready for review

**Comment by muuki88** [comment]
`scalamft` fails. Can you run with `sbt sbt:scalafmt` and commit :smile:

**Comment by sullis** [comment]
obsolete

**Pull request closed without merging**